core/src/rpc.ts

```diff
@@ -23,8 +25,8 @@ const parseInfuraProjectIdFromRpcUrl = function (rpcUrl: string): string | undef
 };
 
 export const setupRpcUrlAndGetNetworks = async function (
-    rpcUrl?: string,
-    isDev?: boolean
+    rpcUrl: string = LOCAL_RPC_URL,
```

Comment:
This function is not only used for test, so it's misleading to hardcode `LOCAL_RPC_URL` here. Core functionality is used without env variable `LOCAL_RPC_URL` whatsoever, so this env variable shouldn't appear in the regular (non-test-related) core functions